dot/rpc/subscription/websocket_integration_test.go

```diff
@@ -18,186 +18,415 @@ import (
 	"github.com/ChainSafe/gossamer/lib/runtime"
 	"github.com/ChainSafe/gossamer/lib/transaction"
 	"github.com/ChainSafe/gossamer/pkg/scale"
-	"go.uber.org/mock/gomock"
 
 	"github.com/gorilla/websocket"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
 )
 
-func TestWSConn_HandleConn(t *testing.T) {
-	ctrl := gomock.NewController(t)
+func TestWSConn_HandleConnParallel(t *testing.T) {
```

Comment:
This function is testing only the method `initStorageChangeListener` so I would suggest to renaming the test func
```suggestion
func TestWSConn_initStorageChangeListener(t *testing.T) {
```